### PR TITLE
Initial docs for ovirt/RHV UPI

### PR DESCRIPTION
**Pull request by gekorob**
Docs to document step by step the basic UPI installation process

https://issues.redhat.com/browse/OCPRHV-280

Enhancement: https://github.com/openshift/enhancements/pull/326

Signed-off-by: Roberto Ciatti <rciatti@redhat.com>

### CODE REVIEW

**Comment by dougsland** [comment]
Hi Roberto,

If am not wrong, title format is: Bug XXXX: Initial docs for ovirt/RHV UPI

> Docs to document step by step the basic UPI installation process
> 
> https://issues.redhat.com/browse/OCPRHV-280
 
I would remove this one ^.

**Comment by dougsland** [comment]
I think would be nice to include this UPI support in the main README file as the others supported platform. :)

**Comment by gekorob** [comment]
Agree, that could be another PR

File: docs/user/ovirt/install_upi.md [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+  `etcd-<idx>.<cluster_name>.<base_domain>` must be added to point to `<idx>`'th control plane machine (internal resolution).
+
+  For each control plane machine a SRV type DNS record must be provided, like:
+
+    ```plain
+    # _service._proto.name.                            TTL   class SRV priority weight port target.
+    _etcd-server-ssl._tcp.$cluster_name.$base_domain   86400 IN    SRV 0        10     2380 etcd-0.$cluster_name.$base_domain.
+    _etcd-server-ssl._tcp.$cluster_name.$base_domain   86400 IN    SRV 0        10     2380 etcd-1.$cluster_name.$base_domain.
+    _etcd-server-ssl._tcp.$cluster_name.$base_domain   86400 IN    SRV 0        10     2380 etcd-2.$cluster_name.$base_domain.
+    ```
```
Comment (sdodson) [inline docs/user/ovirt/install_upi.md:216]: This should no longer be necessary, can you test with out this and remove this?

Comment (gekorob) [reply docs/user/ovirt/install_upi.md]: Thx @sdodson, tested without and working.
Updated the documentation

**Pull request merged**